### PR TITLE
Add activation height for multisig fee sorting

### PR DESCRIPTION
**Pull request by fassadlr**

### CODE REVIEW

File: src/Stratis.Sidechains.Networks/CirrusMain.cs [hunk 1]
```diff
@@ -185,7 +185,8 @@ public CirrusMain()
                 GetMiningTimestampV2ActivationHeight = 3_709_000, // Monday 14 February 00:00:00 (Estimated)
                 GetMiningTimestampV2ActivationStrictHeight = 3_783_000, // Monday 28 February 07:00:00 (London Time) (Estimated)
                 ContractSerializerV2ActivationHeight = 3_386_335, // Monday 13 December 16:00:00 (Estimated)
-                Release1300ActivationHeight = 4_334_400
+                Release1300ActivationHeight = 4_334_400,
+                Release1400ActivationHeight = 0,
```
Comment (fassadlr) [inline src/Stratis.Sidechains.Networks/CirrusMain.cs:189]: I will set this once merged.

**Comment by fassadlr** [comment]
Here is the CI for this

https://dev.azure.com/StratisProject/StratisFullNode/_build/results?buildId=38412&view=results

**Comment by quantumagi** [comment]
CI is failing

**Review by quantumagi — DISMISSED** [review]
LGTM. Merge once CI passes. We can hard code the other activation heights now too - i.e. drop the BIP 9 setups. Also, perhaps we can use a dictionary (or array) for hard-coded activation heights instead of adding a variables each time - e.g. `ActivationHeight["Release1400"]` and `ActivationHeight["ContractSerializerV2"]` OR `ActivationHeight[Release1400]` and `ActivationHeight[ContractSerializerV2]`, where the indexes are defined constants. Just a thought.

**Comment by fassadlr** [comment]
@quantumagi I've ran all the failed tests locally and they pass bar the `EnableAutoKickSync` test.

**Comment by fassadlr** [comment]
> LGTM. Merge once CI passes. We can hard code the other activation heights now too - i.e. drop the BIP 9 setups. Also, perhaps we can use a dictionary (or array) for hard-coded activation heights instead of adding a variables each time - e.g. `ActivationHeight["Release1400"]` and `ActivationHeight["ContractSerializerV2"]` OR `ActivationHeight[Release1400]` and `ActivationHeight[ContractSerializerV2]`, where the indexes are defined constants. Just a thought.

Can you please do a PR where we remove the second BIP 9?

**Pull request merged**